### PR TITLE
Update javaagent download link

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: content/en/docs/instrumentation/java/automatic/_index.md [hunk 1]
```diff
@@ -83,6 +83,6 @@ might want to [annotate](annotations) selected methods or add [manual
 instrumentation](../manual) to collect custom telemetry data.
 
 [Agent Configuration]: agent-config
-[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar
+[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.17.0/opentelemetry-javaagent.jar
```
Comment (svrnm) [inline content/en/docs/instrumentation/java/automatic/_index.md:86]: For the [getting started](https://opentelemetry.io/docs/instrumentation/java/getting-started/) we move to the latest link github provides, would that work as well?

`https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar`

Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/automatic/_index.md:86]:
```suggestion
[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
```

Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/automatic/_index.md:86]:
```suggestion
[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar

Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/automatic/_index.md:86]:
```suggestion
[opentelemetry-javaagent.jar]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
```

Comment (mateuszrzeszutek) [reply content/en/docs/instrumentation/java/automatic/_index.md:86]: Good idea, will do.

**Comment by mateuszrzeszutek** [comment]
Weird, I can't seem to edit this PR in any way -- neither in the browser, nor push my changes. I'll close it and make another one.

**Pull request closed without merging**